app/templates/settings/applications.hbs

```diff
@@ -0,0 +1 @@
+{{settings/application-page}}
```

Comment:
Let's follow a uniform naming schema. 
We'll call this component `settings/applications-section`